client/jetpack-onboarding/steps/business-address.jsx

```diff
@@ -76,14 +78,15 @@ class JetpackOnboardingBusinessAddressStep extends React.PureComponent {
 							<FormFieldset key={ fieldName }>
 								<FormLabel htmlFor={ fieldName }>{ fieldLabel }</FormLabel>
 								<FormTextInput
+									autoFocus={ fieldName === 'name' }
 									id={ fieldName }
 									onChange={ this.getChangeHandler( fieldName ) }
+									required={ fieldName === 'name' }
```

Comment:
The original JPO plugin had the following fields required: business name, street address, city, zip (https://github.com/Automattic/jetpack-onboarding/blob/master/client/components/steps/business-address.jsx#L66). Maybe we should follow that example for the v1.

Reply:
This makes all of them required :) Adapted.